README.md

```diff
@@ -1 +1,48 @@
-# azure-sso-demo
\ No newline at end of file
+# azure-sso-demo
+
+This repository contains reference configurations that can create resourses on Azure using CLI and deploy cross-dc-rhsso-demo on Azure with all the required configurations.
+
+The final architecture looks like:
+
+![Architecture diagram](./scenario.jpg)
+
+
+## Prerequisites
+
+* rhel 8 with registered subscription
+* ansible >= 2.9 / python >= 3.9
+* `ansible-galaxy collection install -r requirements.yml`
+* python3-netaddr installed on the controller host `dnf install python3-netaddr`
+* Azure CLI installed on the controller host 'dnf install azure-cli'
+* All the default paramerts for Azure cloud instances are provided in 'roles/azure/defaults/main.yml'
+* We are using dynamic inventory provided  by Azure. Once the instances are created you can view it by "ansible-inventory -i inventory/myazure_rm.yml --graph"
+* We have provided two options, we can either go with managed mariadb on Azure or we can create the RHEL instances and configuire mariadb using 'mariadb' role on cross-dc-rhsso-demo.
```

Comment:
Break out into a "Database Options" bulletpoint

Reply:
Added:

### Database options

* We have provided two options: 
    1. For managed mariadb on Azure set managed_mariadb as true in `create-demo-setup.yml`.
    2. For unmanaged mariadb on Azure set managed_mariadb as false in `create-demo-setup.yml`. This will create the RHEL instances and configure mariadb using 'mariadb' role on cross-dc-rhsso-demo.